compiler_gym/service/proto/compiler_gym_service.proto

```diff
@@ -350,7 +351,9 @@ message AddBenchmarkReply {}
 
 // An arbitrary string parameter value for a session.
 message SessionParameter {
+  // A string key.
```

Comment:
I love these docs! ;)

Reply:
Oh shush you 😃

These pointless comment strings are actually pretty useful in the generated html docs where there isn't as much context / type info.